tests/unit/v1/_test_cross_language.py

```diff
@@ -137,55 +137,52 @@ def _run_testcase(testcase, call, firestore_api, client):
 def test_create_testprotos(test_proto):
     testcase = test_proto.create
     firestore_api = _mock_firestore_api()
-    client, document = _make_client_document(firestore_api, testcase)
+    client, doc = _make_client_document(firestore_api, testcase)
     data = convert_data(json.loads(testcase.json_data))
-    call = functools.partial(document.create, data)
+    call = functools.partial(doc.create, data)
     _run_testcase(testcase, call, firestore_api, client)
 
 
 @pytest.mark.parametrize("test_proto", _GET_TESTPROTOS)
 def test_get_testprotos(test_proto):
     testcase = test_proto.get
     firestore_api = mock.Mock(spec=["get_document"])
-    response = document_pb2.Document()
+    response = document.Document()
     firestore_api.get_document.return_value = response
-    client, document = _make_client_document(firestore_api, testcase)
+    client, doc = _make_client_document(firestore_api, testcase)
```

Comment:
Having a `document` module and a `doc` variable seems a bit confusing to me, though not terribly so. I'm inclined to say its probably fine as this is a test, but is it not possible import `Document` from the `document` module?

I am also definitely not a Python expert, so its entirely possible this is the Pythonic solution.

Reply:
This was to avoid shadowing. we used to have `document_pb2` and `document` so I moved to `document

1. 

` and `doc` respectively.